r/R/record-batch.R

```diff
@@ -123,7 +123,7 @@ RecordBatch <- R6Class("RecordBatch",
     },
     invalidate = function() {
       .Call(`_arrow_RecordBatch__Reset`, self)
-      super$invalidate()
+      assign(".:xp:.", NULL, envir = self)
```

Comment:
Does this need to be here, or does the call to what was previously `super$invalidate()` just need removing entirely?

Reply:
I think you just want to remove it all: these methods, the one place they're still called, and the C++ functions they call. Since we're doing ALTREP now, it doesn't make sense to invalidate the Arrow data.

Reply:
Done